zfit/core/basemodel.py

```diff
@@ -755,8 +755,18 @@ def sample(self, n: int, limits: ztyping.LimitsType, name: str = "sample") -> zt
         Returns:
             Tensor(n_obs, n_samples)
         """
+        if fixed_params is True:
```

Comment:
I don't like `fixed_params`, maybe better `freeze_params`. @rsilvaco?

Reply:
There's more to that: what about randomization? Shouldn't the data have something like: Data.sample(randomize={True, list_with_params})

Reply:
I'm not sure I follow, can you give me an example of `randomize`?

Reply:
not sure if I understood this point either. You want to keep some parameter completely fixed in the sampling?
For @apuignav, I don't have a strong preference on the name. Why you have a preference for `freeze_params`?